src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsTrackingOptInRequestsOasis.java

```diff
@@ -20,326 +20,326 @@
+//
+//    modal.verifyTrackingRequestsNotSend(urlPatterns, networkTrafficInterceptor);
+//  }
+//
+//  @NetworkTrafficDump(useMITM = true)
+//  @Execute(trackingOptIn = false)
+//  @Test(
+//      dataProviderClass = TrackingOptInDataProvider.class,
+//      dataProvider = "adsQuantcastDataProvider",
+//      groups = "AdsOptInRejectedOasis"
+//  )
+//  public void adsTrackingRejectedForQuantcast(List<String> urlPatterns) {
+//    networkTrafficInterceptor.startIntercepting();
+//    TrackingOptInModal modal = new TrackingOptInModal();
+//    modal.rejectOptInModal(driver, POLAND, ADS_HOME_PAGE);
+//
+//    modal.verifyTrackingRequestsNotSend(urlPatterns, networkTrafficInterceptor);
+//
+//  }
+//
+//  @NetworkTrafficDump(useMITM = true)
+//  @Execute(trackingOptIn = false)
+//  @Test(
+//      dataProviderClass = TrackingOptInDataProvider.class,
+//      dataProvider = "adsComscoreDataProvider",
+//      groups = "AdsOptInRejectedOasis"
+//  )
+//  public void adsTrackingRejectedForComscore(List<String> urlPatterns) {
+//    networkTrafficInterceptor.startIntercepting();
+//    TrackingOptInModal modal = new TrackingOptInModal();
+//    modal.rejectOptInModal(driver, POLAND, ADS_HOME_PAGE);
+//
+//    modal.verifyTrackingRequestsNotSend(urlPatterns, networkTrafficInterceptor);
+//
+//  }
+//
+//  @NetworkTrafficDump(useMITM = true)
+//  @Execute(trackingOptIn = false)
+//  @Test(
+//      dataProviderClass = TrackingOptInDataProvider.class,
+//      dataProvider = "adsGoogleAnalyticsDataProvider",
+//      groups = "AdsOptInRejectedOasis"
+//  )
+//  public void adsTrackingRejectedForGoogleAnalytics(List<String> urlPatterns) {
+//    networkTrafficInterceptor.startIntercepting();
+//    TrackingOptInModal modal = new TrackingOptInModal();
+//    modal.rejectOptInModal(driver, POLAND, ADS_HOME_PAGE);
+//
+//    modal.verifyTrackingRequestsSend(urlPatterns, networkTrafficInterceptor);
+//  }
+//
+//  @NetworkTrafficDump(useMITM = true)
+//  @Execute(trackingOptIn = false)
+//  @Test(
+//      dataProviderClass = TrackingOptInDataProvider.class,
+//      dataProvider = "adsNpaHiViDataProviderOasis",
+//      groups = "AdsOptInRejectedOasis"
+//  )
+//  public void adsTrackingRejectedForHiViAdSlots(List<String> urlPatterns) {
+//    networkTrafficInterceptor.startIntercepting();
+//    TrackingOptInModal modal = new TrackingOptInModal();
+//    modal.rejectOptInModal(driver, POLAND, ADS_HIVI_PAGE);
+//    modal.scrollTo(driver.findElement(By.className("wds-global-footer__header")));
+//
+//    modal.verifyTrackingRequestsSend(urlPatterns, networkTrafficInterceptor);
+//  }
+//
+//  @NetworkTrafficDump(useMITM = true)
+//  @Execute(trackingOptIn = false)
+//  @Test(
+//      dataProviderClass = TrackingOptInDataProvider.class,
+//      dataProvider = "adsPetametricsDataProvider",
+//      groups = "AdsOptInRejectedOasis"
+//  )
+//  public void adsTrackingRejectForPetametrics(List<String> urlPatterns) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)